Automattic/wp-calypso
# Add feature summary strings for Akismet Products

Add summary feature strings for Akismet Products

Related to #

## Proposed Changes

* Add several helper functions for determining type of Akismet product
* Check for Akismet products in the checkout summary component and render strings if available

## Testing Instructions

1. Checkout these changes
2. Start your local environment
3. Go to the following URLs and make sure the feature strings make sense

http://calypso.localhost:3000/checkout/akismet/ak_free_yearly
http://calypso.localhost:3000/checkout/akismet/ak_personal_yearly
http://calypso.localhost:3000/checkout/akismet/ak_plus_yearly_1
http://calypso.localhost:3000/checkout/akismet/ak_plus_monthly_2
http://calypso.localhost:3000/checkout/akismet/ak_plus_yearly_3
http://calypso.localhost:3000/checkout/akismet/ak_plus_monthly_4
http://calypso.localhost:3000/checkout/akismet/ak_ent_yearly_1

Example image ( ak_plus_yearly_3 )

![image](https://user-images.githubusercontent.com/65001528/233497461-7a1ba5ec-d4c4-409d-aa30-d32415ba1a2d.png)


## Pre-merge Checklist
